### PR TITLE
Improve docs and code style

### PR DESCRIPTION
**Pull request by lexmag**

### CODE REVIEW

File: lib/decimal.ex [hunk 1]
```diff
@@ -70,8 +72,10 @@ defmodule Decimal do
   @type exponent :: integer
 
   @typedoc """
+
     * `1` for positive
     * `-1` for negative
+
```
Comment (ericmj) [inline lib/decimal.ex]: Looks like we don't have this space and the one above anywhere else.

File: lib/decimal.ex [hunk 2]
```diff
@@ -853,26 +873,31 @@ defmodule Decimal do
   A decimal number will always be created exactly as specified with all digits
   kept - it will not be rounded with the context.
 
-  ## BNFC
+  ## BNF Converter
```
Comment (ericmj) [inline lib/decimal.ex]: Should probably just be "BNF".

File: lib/decimal.ex [hunk 3]
```diff
@@ -853,26 +873,31 @@ defmodule Decimal do
...
-  def new(float) when is_float(float),
-    do: float |> :io_lib_format.fwrite_g |> IO.iodata_to_binary |> new
+  def new(%Decimal{} = num), do: num
+
+  def new(int) when is_integer(int) do
+    %Decimal{sign: (if int < 0, do: -1, else: 1), coef: Kernel.abs(int)}
+  end
+
+  def new(float) when is_float(float) do
+    float |> :io_lib_format.fwrite_g |> IO.iodata_to_binary |> new
```
Comment (ericmj) [inline lib/decimal.ex]: Parenthesis? :bowtie:

File: lib/decimal.ex (hunk 1 above)
Comment (ericmj) [inline lib/decimal.ex]: Actually, I see that you added this almost everywhere. I don't think the trailing whitespace line adds any value but if we should have it we should probably add it to the style guide?

Comment (whatyouhide) [inline lib/decimal.ex]: @ericmj we should add it to a "documentation" style guide which still needs to happen 😄

**Comment by lexmag** [comment]
Updated.

File: lib/decimal.ex [hunk 4]
```diff
@@ -70,8 +72,10 @@ defmodule Decimal do
   @type exponent :: integer
 
   @typedoc """
+
```
Comment (ericmj) [inline lib/decimal.ex]: ✂️

**Comment by ericmj** [comment]
Thank you!

**Pull request merged**